src/index.html

```diff
@@ -1,14 +1,72 @@
 <!doctype html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta
+    charset="UTF-8">
+    <meta
+    name="viewport"
+          content="width=device-width, user-scalable=no,
+          initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
+    <meta
+    http-equiv="X-UA-Compatible"
+    content="ie=edge">
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <link
+    rel="stylesheet"
+    href="./style.css">
+    <link
+    rel="preconnect"
+    href="https://fonts.googleapis.com"
+    >
+    <link
+    rel="preconnect"
+    href="https://fonts.gstatic.com"
+    crossorigin="anonymous "
+    >
+    <link
+    href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+    rel="stylesheet"
+    >
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#" class="logo-link">
+        <img
+      class="logo"
+      src="images/logo.png"
+      alt="moyo-logo"
+      >
+      </a>
+      <nav class="navigation">
+        <ul class="navigation--list">
+          <li class="nav-item">
+            <a class="nav-item-link is-active" href="#">Apple</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-item-link" href="#Samsung">Samsung</a>
```

Comment:
`href` attribute is typically lowercased, it doesn't matter in this particular case, but I would keep that in mind, when you'll start working with HTTP and routing